index.js

```diff
@@ -45,6 +48,7 @@ Toolkit.run(async tools => {
       ['version', '--allow-same-version=true', '--git-tag-version=false', current])
     console.log('current:', current, '/', 'version:', version)
     newVersion = execSync(`npm version --git-tag-version=false ${version}`).toString().trim()
+    newVersion = `${tagPrefix}${newVersion}`
```

Comment:
use:
```
newVersion = `${process.env['INPUT_TAG-PREFIX']}${newVersion}`
```
and remove `@actions/core` from dependency => speed

Reply:
fixed